rustls/src/client/tls12.rs

```diff
@@ -436,7 +437,7 @@ impl State<ClientConnectionData> for ExpectServerKx<'_> {
         )?;
         self.transcript.add_message(&m);
 
-        let ecdhe = opaque_kx
+        let kx = opaque_kx
```

Comment:
Would you like me to make a separate commit for this change?

Reply:
Yes, it would be nice to keep renaming and other changes that are not strictly tied to the FFDHE changes in a separate commit that comes before the main FFDHE commit.

Reply:
Will do.